### PR TITLE
Dox 261 2 22 release notes

### PR DESCRIPTION
**Pull request by rsilvery**
* 2.22 release notes
* Updates with new labels to previous release notes
* Consent API docs

### CODE REVIEW

File: pages/dev-docs/consent/consent-api.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## Endpoints
+Below, you'll find the list of endpoints that have been activated to facilitate developer access to the consent APIs.
+
+### GET privacy experiences
+- **Request**: GET http://localhost:8080/api/v1/privacy-experience
+- **Description**:  Fetches all the privacy experiences across all the regions using query params to filter the response to the desired response.
+- **Query params**:
+    - `component=overlay`: Instructs the backend to return an overlay type (either regular or TCF).
+    - `region=eea`: a query param to further filter the experiences by region. 
+    - `fides_user_device_id=<fides user device id>`: used to supplement the experience with any previously saved preferences for the specified user.
```
Comment (pattisdr) [inline pages/dev-docs/consent/consent-api.mdx:21]: Maybe we actually remove this query param, I know I had it in my internal docs, but customers wouldn't be using this one in the developer-friendly consent API's context

File: pages/dev-docs/consent/consent-api.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  'http://localhost:8080/api/v1/privacy-preferences' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "browser_identity": {
+    "fides_user_device_id": "f54b192b-6b97-4312-b5f5-4608eb8f0558"
+  },
+  "fides_string": "CPzHq4APzHq4AAMABBENAUEAALAAAEOAAAAAAEAEACACAAAA.IAEAEAAA"
+}'
+````
```
Comment (pattisdr) [inline pages/dev-docs/consent/consent-api.mdx:55]: Do we also want to include the new endpoint added here: http://localhost:8080/api/v1/plus/tcf/decode_for_mobile?fides_string=?<tc_string>

https://ethyca.atlassian.net/wiki/spaces/EN/pages/2805006351/2023-10-04+Developer-Friendly+Consent+API+Documentation#GET-Decode-TC-String-Developer-Friendly-Endpoint-(Coming-in-2.22.0)

File: pages/dev-docs/consent/consent-api.mdx (hunk 1 above)
Comment (rsilvery) [reply pages/dev-docs/consent/consent-api.mdx:21]: Sorry, which parameter?

File: pages/dev-docs/consent/consent-api.mdx (hunk 2 above)
Comment (rsilvery) [reply pages/dev-docs/consent/consent-api.mdx:55]: Check now please @pattisdr

File: pages/dev-docs/consent/consent-api.mdx (hunk 1 above)
Comment (pattisdr) [reply pages/dev-docs/consent/consent-api.mdx:21]: ah sorry the line on which I commented: `fides_user_device_id`

Comment (rsilvery) [reply pages/dev-docs/consent/consent-api.mdx:21]: I need to get these out so the link isn't broken in the announcement but I'll fix this in another branch

**Pull request merged**